.github/workflows/WorkitemValidation.yaml

```
@@ -15,6 +15,7 @@ defaults:
 
 jobs:
   GitHubIssueValidation:
+    if: github.event.pull_request.head.repo.full_name == 'microsoft/BCApps'
```

Wait.. what's the head/base repo in thi case? Isn't the head repo where the PR is coming from. I.e. if someone opens up a PR into microsoft/BCApps from a fork, the head repo is user/BCApps and the base repo is microsoft/BCApps?

```suggestion
    if: github.event.pull_request.base.repo.full_name == 'microsoft/BCApps'
```